code/modules/food/recipes_microwave.dm

```diff
@@ -813,7 +813,7 @@
 	result = /obj/item/weapon/reagent_containers/food/snacks/meatsteak/synth
 
 /datum/recipe/meatsteak
-	reagents = list(SODIUMCHLORIDE = 1, BLACKPEPPER = 1)
+	// reagents = list(SODIUMCHLORIDE = 1, BLACKPEPPER = 1)
```

Comment:
you can't just change an extremely commonly made recipe with no cl

Reply:
That was only for testing purposes and wouldn't be in the finished PR.